src/__tests__/createActions-test.js

```diff
@@ -0,0 +1,212 @@
+import { createActions } from '../';
+import { isFSA } from 'flux-standard-action';
+
+describe('createActions', () => {
+  describe('createActions(actionMap)', () => {
+    it('returns an object with actions for each key', () => {
+      const a = createActions({
+        one: (b, c) => ({ bat: b, cat: c }),
+        two: {
+          payload: (d, e) => ({ dog: d, egg: e }),
+          meta: (/* d, e */) => ({ zed: 1 })
+        },
+        three: {
+          meta: (/* f */) => ({ zed: 2 })
+        }
+      });
+      expect(a).to.be.instanceOf(Object);
+      const a1 = a.one(1, 2);
+      expect(a1).to.eql({ type: 'one', payload: { bat: 1, cat: 2 } });
+      expect(isFSA(a1)).to.be.true;
+      expect(a.one.toString()).to.equal('one');
+      const a2 = a.two(3, 4);
+      expect(a2).to.eql({
+        type: 'two',
+        payload: { dog: 3, egg: 4 },
+        meta: { zed: 1 }
+      });
+      expect(isFSA(a2)).to.be.true;
+      expect(a.two.toString()).to.equal('two');
+      const a3 = a.three('foo');
+      expect(a3).to.eql({
+        type: 'three',
+        payload: 'foo',
+        meta: { zed: 2 }
+      });
+      expect(isFSA(a3)).to.be.true;
+      expect(a.three.toString()).to.equal('three');
+    });
+  });
+
+  describe('createActions(actionMap, arrDefTypes)', () => {
+    it('returns an object with actions and default actions', () => {
+      const a = createActions(
+        {
+          one: (b, c) => ({ bat: b, cat: c }),
+          two: {
+            payload: (d, e) => ({ dog: d, egg: e }),
+            meta: (/* d, e */) => ({ zed: 1 })
+          },
+          three: {
+            meta: (/* f */) => ({ zed: 2 })
+          }
+        }, [
+          'four',
+          'five'
+        ]
+      );
+      expect(a).to.be.instanceOf(Object);
```

Comment:
Does it matter that this is an instance of an `Object`?

Maybe testing for plain object would be better, since the space of things that are Objects is largely than that of what's being tested.

```js
new Function() instanceof Object // true
[] instanceof Object // true
```

In general, I would advocate removing this assertion, since in a loosely-typed language like JavaScript, I'm only ever going to care that I can call `a.one(...)`, `a.two(...)`, etc. 

That is, I shouldn't care about the type of a thing, but only about what messages that thing responds to. Said yet another way, I don't care that it's a duck, I only care that it quacks.